### ISSUE TITLE
Error while starting Elastalert

### ISSUE THREAD

**Issue by mancharagopan** [bitsensor/elastalert#67]
I am using npm. when i start elastalert i am receiving following error.

ProcessController:  /usr/bin/python: No module named elastalert

**Comment by mancharagopan** [comment]
I change the working directory in config.json file and installed all dependencies. Now i am getting a new error. Can somebody help?

```
[root@localhost elastalert]# npm start

> @bitsensor/elastalert@1.0.0 start /root/elastalert
> sh ./scripts/start.sh

02:47:56.190Z  INFO elastalert-server: Config:  No config.dev.json file was found in /root/elastalert/config/config.dev.json.
02:47:56.193Z  INFO elastalert-server: Config:  Proceeding to look for normal config file.
02:47:56.202Z  INFO elastalert-server: Config:  A config file was found in /root/elastalert/config/config.json. Using that config.
02:47:56.221Z  INFO elastalert-server: Router:  Listening for GET request on /.
02:47:56.221Z  INFO elastalert-server: Router:  Listening for GET request on /status.
02:47:56.221Z  INFO elastalert-server: Router:  Listening for GET request on /status/control/:action.
02:47:56.221Z  INFO elastalert-server: Router:  Listening for GET request on /status/errors.
02:47:56.221Z  INFO elastalert-server: Router:  Listening for GET request on /rules.
02:47:56.223Z  INFO elastalert-server: Router:  Listening for GET request on /rules/:id.
02:47:56.223Z  INFO elastalert-server: Router:  Listening for POST request on /rules/:id.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for DELETE request on /rules/:id.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for GET request on /templates.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for GET request on /templates/:id.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for POST request on /templates/:id.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for DELETE request on /templates/:id.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for POST request on /test.
02:47:56.224Z  INFO elastalert-server: Router:  Listening for GET request on /config.
02:47:56.225Z  INFO elastalert-server: Router:  Listening for POST request on /config.
02:47:56.225Z  INFO elastalert-server: Router:  Listening for POST request on /download.
02:47:56.225Z  INFO elastalert-server: Router:  Listening for GET request on /metadata/:type.
02:47:56.225Z  INFO elastalert-server: Router:  Listening for GET request on /mapping/:index.
02:47:56.239Z  INFO elastalert-server: ProcessController:  Starting ElastAlert
02:47:56.239Z  INFO elastalert-server: ProcessController:  Creating index
02:47:56.265Z ERROR elastalert-server:
    Server:  Starting server failed with error: TypeError: Cannot read property 'toString' of null
        at ProcessController.start (/root/elastalert/src/controllers/process/index.js:46:28)
        at /root/elastalert/src/elastalert_server.js:71:33
        at /root/elastalert/src/common/config/server_config.js:60:9
        at Array.forEach (<anonymous>)
        at /root/elastalert/src/common/config/server_config.js:59:22
        at <anonymous>
02:47:56.265Z  INFO elastalert-server: Server:  Stopping server
02:47:56.265Z  INFO elastalert-server: ProcessController:  ElastAlert is not running
02:47:56.265Z  INFO elastalert-server: Server:  Server stopped. Bye!
```

**Comment by Theoooooo** [comment]
I got the same error. Still waiting for an answer

**Comment by wynnchel** [comment]
I got the same problem with even the same stacktrace.

**Comment by datnguyen19941** [comment]
I have the same error. Someone have solution to fix it

**Comment by amccrea** [comment]
Same

**Comment by martijnrondeel** [comment]
This should be fixed in [3.0.0-beta.0](https://github.com/bitsensor/elastalert/releases/tag/3.0.0-beta.0).
